### ISSUE TITLE
Can I keep the "arrow" button, but hide the "done" button?

### ISSUE THREAD

**Issue by IgorGiroti** [AppIntro/AppIntro#938]
I want keep the "arrow" in the DotIndicator (Image Bellow), but when I reach the last slide, I don't want that the done button appears. There is a config where I can do this? If I set isButtonsEnabled = false, the Arrow disappears too.

![image](https://user-images.githubusercontent.com/70269019/118009205-3ee01700-b324-11eb-8b91-2e090bd440ab.png)

**AppIntro Version**: 6.1.0

Thanks!

**Comment by IgorGiroti** [comment]
Update: I "solved" the problem by creating my own custom button and disabling the lib buttons, but I leave my suggestion for creating a "customization class" in which we can change the position of the indicator and buttons.

**Comment by cortinico** [comment]
> I don't want that the done button appears. There is a config where I can do this? If I set isButtonsEnabled = false, the Arrow disappears too.

Can you describe your use case a bit more? Why the done button doesn't fit your scenario?

**Comment by IgorGiroti** [comment]
Yes, I want use the indicator in the center of the screen, and just showing the arrows (left or right). It's more a design preference.

**Comment by cortinico** [comment]
> and just showing the arrows (left and right). It's more a design preference.

Have you considered using the `wizard` mode?

> The "done button" I will implement in the corner of the screen, where the user can click there on any page.

This can be achieved with a custom layout

**Comment by IgorGiroti** [comment]
Using the custom layout I have only achieved move indicators, not the arrows or "done button". About the wizard mode, I'll check a little more.

**Comment by cortinico** [comment]
> Using the custom layout I have only achieved move indicators, not the arrows or "done button".

The custom layout gives you freedom to whatever is above the arrows and the buttons. So if you plan to add a cross (X) button on every slide to skip the intro, say in the top right corner, you can do it with a custom layout.
